### PR TITLE
unit tests for static functions + packages restructuring

### PR DESCRIPTION
**Pull request by gu1ll0me**
-Tried to do my best to adapt the module for future unit tests.
-Restructured the module into packages (not sure for this one, tell me if i'm wrong) following this doc : http://docs.python.org/3/tutorial/modules.html#packages
-README renaming took from this: http://guide.python-distribute.org/creation.html

### CODE REVIEW

File: ccr/session.py [hunk 1]
```diff
@@ -152,7 +71,7 @@ def __init__(self, username, password, rememberme=False):
             "system": 16,
             "x11": 17,
             "utils": 18,
-            "lib32": 19,
+            "lib32": 19
```
Comment (Inkane) [inline ccr/session.py]: In general, it is useful to have a comma after the last element of lists/maps, as git diff will understand it better when you insert a new line later.

File: ccr/session.py [hunk 2]
```diff
@@ -419,7 +338,7 @@ def submit(self, f, category):
         error = re.compile(r"<span class='error'>(?P<message>.*)</span>")
         data = {
             "pkgsubmit": 1,
-            "category": self._cat2number[category],
```
Comment (Inkane) [inline ccr/session.py:422]: same here

File: ccrcli-test.py [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
-    print("Version      : %s" % r.Version)
-    print("URL          : %s" % r.URL)
-    print("License      : %s" % r.License)
-    print("Category     : %s" % r.Category)
-    print("Maintainer   : %s" % r.Maintainer)
-    print("Description  : %s" % r.Description)
-    print("OutOfDate    : %s" % r.OutOfDate)
-    print("Votes        : %s" % r.NumVotes)
-    print("Screenshot   : %s" % r.Screenshot)
+    # print("Name         : %s" % r.Name)
```
Comment (Inkane) [inline ccrcli-test.py]: If this is dead code, it should be removed. Thank to git, we can still restore if it is needed at some point in the future.

File: tests/testccrstatic.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import inspect
+import unittest
```
Comment (Inkane) [inline tests/testccrstatic.py:2]: This looks generally good. You might want to explore mock testing, though (http://www.voidspace.org.uk/python/mock/)

**Comment by Inkane** [comment]
I'm generally happy with the refactoring, though please comment on my nitpicks before merging.

File: ccr/session.py (hunk 1 above)
Comment (gu1ll0me) [inline ccr/session.py]: Noted. I understood the purpose with this explanation: http://stackoverflow.com/a/11597911

File: ccrcli-test.py (hunk 3 above)
Comment (gu1ll0me) [inline ccrcli-test.py]: Indeed. The info function code is tested with testccrstatic.py anyway.

**Comment by gu1ll0me** [comment]
Last commit should be good. I will merge if there is no more fixes. I will continue with unit tests for the session class and set up a mock for the ccr database.

**Pull request merged**